tests/test_items.py

```diff
@@ -43,18 +46,48 @@ def test_new_item_history():
     assert item_history.item_id == item_id
 
 
-@pytest.mark.skip(reason="Item.history() not implemented")
+
 def test_item_assign():
     """ Test item assign """
+
+    company = Company(
+        id=223,
+        name="Bad Company",
+        code="Bad Cmpny",
+        address="addr"
+    )
+
+    location = Location(
+        id=104,
+        name="Test location",
+        code="L",
+        country="Country",
+        region="Region",
+        city="City",
+        type="Type",
+        address="Address",
+        longitude="11223344",
+        latitude="44332211",
+        status="T",
+        company_id=223,
+        poster_id=100,
+        synchronized_on=datetime.utcnow,
+        working_hours=1,
+        closed_days=2
+    )
+
     new_item = Item(
         id=1,
-        name="Nanomedikit",
+        name="Duck Eggs",
         stock_date=datetime.utcnow,
-        comment="Medikit with nano particles",
-        company_id=223
+        comment="Eggs from ducks",
+        company_id=company.id,
+        created_on=datetime.utcnow,
+        updated_on=datetime.utcnow,
+        company=company
     )
 
-    new_employee = Item(
+    new_employee = Employee(
```

Comment:
@paulodamaso I would name this variable ``receiver`` or ``assignee`` (the one who receives or is assigned an Item)